### DISCUSSION TITLE
Upgraded Fief docker image now admin wants me to setup workspace again

### DISCUSSION THREAD

**Discussion by matteius** [fief-dev/fief#198] [Q&A] [answered]
I upgraded the Fief docker image but not the underlying secrets tied to the deployment -- the site continues to function and allow users to login/etc -- but the admin was impacted as I can no longer get to the admin to configure the existing workspace, it wants me to create a new one.  I'm going to keep digging in here, but curious if anything comes to mind, Thanks!

![image](https://github.com/fief-dev/fief/assets/479892/3d7724b5-2b61-4c0a-a8f6-ac05b855ea1f)

**Comment by matteius** [comment, marked as answer]
Ah ok I think I know whats happening (kind-of) -- since I logged in outside the scope of the /admin URL (using my site) then I didn't have context for the admin page, and it wanted me to setup a new workspace (that part confuses me--technically I wouldn't want anyone else to be able to setup a new workspace).

**Reply by ScottFred** [reply]
I agree with you.  I came to the Discussion Forums just now to understand why a non-admin User I created was being asked to create a new Workspace.  This seems like a security flaw to me.  In addition, the UI for this workflow does NOT allow the user to Cancel the "Create Workspace" flow.

**Reply by benabel** [reply]
@matteius @ScottFred I have the same Workspace creation for new users. Have you opened an issue for this instead of a discussion.

**Reply by ScottFred** [reply]
@benabel  I didn't open a new issue because @frankie567 knows that he has to get roles put in place before he can assign users to roles.  I also understood him to say that he admits that he was focused on Fief Cloud and this wouldn't happen in this case.  So, I think it's lower on his priority list.

**Reply by benabel** [reply]
OK thanks for your answer @ScottFred.
